ethsigner/signer/hashicorp/src/test/java/tech/pegasys/ethsigner/signer/hashicorp/HashicorpSubCommandTest.java

```diff
@@ -115,8 +163,28 @@ public void missingOptionalParametersAreSetToDefault() {
         "host", hcConfig::toString, "/secret/data/ethsignerSigningKey");
   }
 
+  @Test
+  void missingClientCertificateFileDisplaysErrorIfPasswordIsStillIncluded() {
+    missingParameterShowsError("tls-truststore-file");
+  }
+
+  @Test
+  void missingClientCertificatePasswordFileDisplaysErrorIfCertificateIsStillIncluded() {
+    missingParameterShowsError("tls-truststore-password-file");
+  }
+
+  @Test
+  void cmdlineIsValidIfBothClientCertAndPasswordAreMissing() {
```

Comment:
```suggestion
  void commandLineIsValidIfBothClientCertAndPasswordAreMissing() {
```

Reply:
Not valid anymore. Resolving conversation.